### PR TITLE
Fixed up file extensions

### PR DESCRIPTION
**Pull request by JaniruTEC**

### CODE REVIEW

**Comment by JaniruTEC** [comment]
Closes https://github.com/cryptomator/cryptomator/issues/1089

File: resources/app/Info.plist [hunk 1]
```diff
@@ -88,6 +88,29 @@
         </array>
       </dict>
     </dict>
+    <dict>
+      <key>UTTypeIdentifier</key>
+      <string>org.cryptomator.encrypted-cryptomator-vault-file</string>
```
Comment (tobihagemann) [inline resources/app/Info.plist]:
```suggestion
      <string>org.cryptomator.encrypted-data</string>
```

File: resources/app/Info.plist [hunk 2]
```diff
@@ -88,6 +88,29 @@
...
+      <key>UTTypeConformsTo</key>
+      <array>
+        <string>public.data</string>
+      </array>
+      <key>UTTypeDescription</key>
+      <string>Encrypted Cryptomator vault file</string>
```
Comment (tobihagemann) [inline resources/app/Info.plist]:
```suggestion
      <string>Cryptomator Encrypted Data</string>
```

File: resources/app/Info.plist [hunk 3]
```diff
@@ -88,6 +88,29 @@
...
+      <key>UTTypeIconFile</key>
+      <string>Cryptomator-Vault.icns</string>
+      <key>UTTypeTagSpecification</key>
+      <dict>
+        <key>public.filename-extension</key>
+        <array>
+          <string>c9r</string>
```
Comment (tobihagemann) [inline resources/app/Info.plist:106]:
```suggestion
          <string>c9r</string>
          <string>c9s</string>
```

File: resources/app/Info.plist [hunk 4]
```diff
@@ -88,6 +88,29 @@
...
+        </array>
+        <key>public.mime-type</key>
+        <array>
+          <string>application/x-vnd.encrypted-cryptomator-vault-file</string>
```
Comment (tobihagemann) [inline resources/app/Info.plist]:
```suggestion
          <string>application/x-vnd.cryptomator.encrypted-data</string>
```

**Review by tobihagemann — CHANGES_REQUESTED** [review]
~~Furthermore, please add another entry in `CFBundleDocumentTypes` for the newly added type identifier.~~ Nope, after further discussion, we don't actually want to open c9r/c9s files with Cryptomator.

File: resources/app/Info.plist (hunk 3 above)
Comment (JaniruTEC) [reply resources/app/Info.plist:106]: GitHub's UI wrongly shows this suggestion to be ignored, when it was indeed added in the commit f5b7007e58698f2c0fe254df331905bc10c2ae0f.

**Review by tobihagemann — APPROVED** [review]

**Pull request merged**